packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -54,7 +54,11 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint256 minParticipationRequirement; // Minimum staked tokens that must vote to resolve a request.
         uint256 minAgreementRequirement; // Minimum staked tokens that must agree on an outcome to resolve a request.
         uint256 cumulativeStakeAtRound; // Total staked tokens at the start of the round.
-        uint64 pendingResolvedIndex; // Index of pendingPriceRequestsIds that has been traversed this round.
+    }
+
+    struct PendingProcessed {
```

Comment:
Is the struct necessary as we only use it once? I'm thinking if it could be simpler to have instead 2 variables like:
```
uint32 public lastRoundIdToProcess;
uint64 public nextIndexToProcess;
```

Reply:
you are totally right, will fix this